src/main/java/bio/terra/service/filedata/exception/ProcessResultSetException.java

```diff
@@ -0,0 +1,7 @@
+package bio.terra.service.filedata.exception;
+
+public class ProcessResultSetException extends RuntimeException {
+  public ProcessResultSetException(String message, java.lang.Throwable cause) {
```

Comment:
The fully qualified name isn't needed here. Also since it's `java.lang` you don't need to import it either.
```suggestion
  public ProcessResultSetException(String message, Throwable cause) {
```